src/service/email.js

```diff
@@ -5,17 +5,29 @@ const path = require('path')
 
 class Email {
 	constructor(host, port, username, password) {
-		const transporter = nodemailer.createTransport({
-			host: host,
-			port: port,
-			secure: true,
-			auth: {
-				user: username,
-				pass: password
-			}
-		})
-
-		this.transporter = transporter
+		if (port === '587' || port === '25') {
```

Comment:
Fixed

Reply:
Still not quite what I ment.

If you revert it to the original code and just replace `secure: true` with: 

```
secure: port === '587' || port === '25'
```

This way we don‘t need any if statements and still get the same functionality with much more clean code.

Reply:
but then secure is true when using port 587 or 25 it must be true of NOT port 587 or 25

Reply:
2 Questions, why AND Operator and why these weird quotes lul

Reply:
We could also just check if the port is 465 as it‘s the only port where secure needs to be set to true.

Reply:
that is true, but e.g. port 2525 is also common for smtp not smtps though. but yes it seems that only port 465 is common for smtps, so using `secure: port ==='465'` seems fine. but for everyone not using the common port for smtps theyll get an error. which could only be fixed be checking the HELO from the smtp protocol to check for tls or ssl

Reply:
The wierd quotes are only because I am typing on mobile, use the same ones as before 😄 

The AND because we only want to set secure to true if the port is not 587 AND not 25

Reply:
The use the check for ports 578 and 25 if those are the only ones where secure should be false.

Reply:
for port 2525 according to the post in the discussion, secure should also be set to false, and port 465 is more fixed to ssl, i think, so i think we should check for port 465. but i could be wrong

Reply:
Hmm okay. I think in Nodemailers docs it is also recommended to only set secure to true for port 465, so lets go with that option.

Reply:
commited